core/chain-configs/src/config.rs

```diff
@@ -0,0 +1,78 @@
+use near_config_utils::ValidationError;
+use near_primitives::num_rational::Rational32;
+use serde_json::Value;
+use std::fs::File;
+use std::io::Read;
+use std::path::Path;
+use crate::genesis_config::GenesisConfigSnapshot;
+use smart_default::SmartDefault;
+
+#[derive(Clone, SmartDefault, serde::Serialize, serde::Deserialize, Debug)]
+pub struct ChainConfig {
```

Comment:
Sure, I will go with the second approach and hope I will come up with a good name

Reply:
@jakmeier, I think that it is better to go with the first approach now since we changed Epoch Id to Protocol Version. Do you agree? I am asking because parameters will not be changing from epoch to epoch

Reply:
Yeah I think it's fine to rename that other struct. I don't really like `InitChainConfig` as a name but can't think of something better right now. Maybe you can come up with something.

Reply:
Sure, I will rename the other struct in the final PR. I will default to the `InitChainConfig` name if I do not come up with a better name when implementing all the needed changes.